sw/device/tests/crypto/cryptotest/firmware/BUILD

```diff
@@ -324,34 +333,5 @@ opentitan_test(
     silicon_owner = silicon_params(
         tags = ["broken"],
     ),
-    deps = [
-        ":aes",
-        ":aes_sca",
-        ":drbg",
-        ":ecdh",
-        ":ecdsa",
-        ":hash",
-        ":hmac",
-        ":ibex_fi",
-        ":kmac_sca",
-        ":otbn_fi",
-        ":prng_sca",
-        ":sha3_sca",
-        ":trigger_sca",
-        "//sw/device/lib/base:csr",
-        "//sw/device/lib/base:status",
-        "//sw/device/lib/crypto/drivers:entropy",
-        "//sw/device/lib/testing/test_framework:check",
-        "//sw/device/lib/testing/test_framework:ottf_main",
-        "//sw/device/lib/testing/test_framework:ujson_ottf",
-        "//sw/device/lib/ujson",
-        "//sw/device/tests/crypto/cryptotest/json:aes_commands",
-        "//sw/device/tests/crypto/cryptotest/json:commands",
-        "//sw/device/tests/crypto/cryptotest/json:drbg_commands",
-        "//sw/device/tests/crypto/cryptotest/json:ecdh_commands",
-        "//sw/device/tests/crypto/cryptotest/json:ecdsa_commands",
```

Comment:
Should we add this to `FIRMWARE_DEPS`? Looks like it will be skipped here.

Reply:
Or we can add it here separately if we don't need as a whole in `FIRMWARE_DEPS`.

Reply:
It was a mistake that `ecdsa_commands` was missing from the `firmware` dependency list before. I tried locally, and it turns out all of the `:*_commands` dependencies are redundant, because they are included by `:commands`, so we can delete all of them.

Reply:
Cool!

Reply:
Done.